CODEOWNERS

```diff
@@ -95,7 +95,7 @@ go.mod @fleetdm/go
 /handbook/digital-experience        @sampfluger88
 /handbook/customer-success          @sampfluger88
 /handbook/demand                    @sampfluger88
-/handbook/engineering               @sampfluger88 @lukeheath
+/handbook/engineering               @lukeheath
```

Comment:
@Sampfluger88 The issue with having us both listed is that I can't merge PRs into the engineering handbook without your approval, even if fleet releaser approves.

Reply:
@lukeheath I believe we fixed this with the changes to maintainers we made the other day. Please LMK if I'm incorrect here. Removing changes to CODEOWNERS for now